hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/replication/VerifyReplication.java

```diff
@@ -736,6 +740,40 @@ private static void printUsage(final String errorMsg) {
     System.err.println(" $ hbase " +
         "org.apache.hadoop.hbase.mapreduce.replication.VerifyReplication" +
         " --starttime=1265875194289 --endtime=1265878794289 5 TestTable ");
+    System.err.println();
+    System.err.println(" To verify the data in TestTable between the cluster runs VerifyReplication and cluster-b");
```

Comment:
nice work to add examples on how to run VerifyReplication between non secure and secure environment. It is very helpful.